recipes/avrocpp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.8.2" %}
+{% set hash = "c17d84613187dda3611c9c95d6f3388036a330ba" %}
+{% set name = "avrocpp" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: avro-cpp-{{ version }}.tar.gz
+    url: https://archive.apache.org/dist/avro/avro-{{ version }}/cpp/avro-cpp-{{ version }}.tar.gz
+    sha1: {{ hash }}
+
+build:
+    number: 0
+    skip: true  # [win]
```

Comment:
It seemed like only `win and py2k` didn't work, but now you're skipping all Windows builds? Is there a reason not to do the newer-platform Windows builds?

If you do, you should probably add <code>python &nbsp;# [win]</code> as a `build` requirement, add [VC features](https://conda-forge.org/docs/buildwin.html#vc-features) to the `build` block, and change it to <code>skip: true &nbsp;# [win and not py36]</code>.